analysis/table1/table1.R

```diff
@@ -60,6 +67,15 @@ print("Create exposure indicator")
 
 df$exposed <- !is.na(df$exp_date_covid)
 
+# Select for pre-existing conditions
+print("Select for pre-existing conditions")
+
+preex_string <- ""
+if (preex != "All") {
+  df <- df[df$sup_bin_preex == preex, ]
+  preex_string <- paste0("_", preex)
```

Comment:
As for the YAML names, we want to know what TRUE/FALSE refers to:
```suggestion
  preex_string <- paste0("preex_", preex)
```